app/code/core/Mage/Adminhtml/etc/adminhtml.xml

```diff
@@ -145,6 +149,9 @@
                                     <blocks translate="title">
                                         <title>Blocks</title>
                                     </blocks>
+                                    <orphaned_resources translate="title">
+                                        <title>Orphaned Resources</title>
```

Comment:
```suggestion
                                        <title>Orphaned ACLs</title>
```

what about this?

Reply:
I changed it to "Orphaned Role Resources" to be consistent to existing label used in the core.